webui/src/pages/repositories/repository/pulls/pullDetails.jsx

```diff
@@ -55,21 +56,26 @@ const PullDetailsContent = ({repo, pull}) => {
             setLoading(false);
             return;
         }
-        try {
-            await pullsAPI.update(repo.id, pull.id, {status: PullStatus.merged});
-            window.location.reload(); // TODO: replace with a more elegant solution
-        } catch (error) {
-            setError(`Failed to update pull-request status: ${error.message}`);
-            setLoading(false);
+        for (let i = 0; i < RETRIES_COUNT; i++) {
```

Comment:
We are already importing https://www.npmjs.com/package/exponential-backoff so lets use it